bugsnag-android-core/src/main/java/com/bugsnag/android/AppDataCollector.kt

```diff
@@ -102,9 +107,19 @@ internal class AppDataCollector(
      *
      * @return the duration in ms
      */
-    internal fun calculateDurationInForeground(): Long? {
+    internal fun calculateDurationInForeground(inForeground: Boolean? = sessionTracker.isInForeground): Long? {
+        if (inForeground == null) return null
```

Comment:
nit: the general (implicit) style rule at Bugsnag is to always use braces for single-line conditionals